cfme/services/myservice/ui.py

```diff
@@ -187,10 +187,7 @@ class ServiceVMDetailsView(VMDetailsEntities):
     @property
     def is_displayed(self):
         return (
-            self.logged_in_as_current_user and
-            self.navigation.currently_selected == ['Services', 'MyServices'] and
-            self.myservice.is_opened and
-            self.title.text == 'VM and Instance "{}"'.format(self.context['object'].name)
+            self.title.text == 'VM and Instance "{}"'.format(self.context['object'].vm_name)
```

Comment:
**Question** Why were these additional checks removed? Have they changed or or unavailable ?

Reply:
These checks not available in this page.